### DISCUSSION TITLE
Built-in formatter ruff doesn't work

### DISCUSSION THREAD

**Discussion by rwxmad** [nvimtools/none-ls.nvim#39] [Q&A] [answered]
With 'local sources = { null_ls.builtins.formatting.ruff }' formatting doesn't work, use 'local sources = { null_ls.builtins.formatting.ruff_format }' instead? Or is there something wrong with the ruff?

**Comment by jakenvac** [comment, marked as answer]
Yep - looks like this is broken & we don't need two builtins of the same tool, either. Thanks for pointing this out.

I've raised an issue #40 and a PR #41. Feel free to take a look and let me know if I've missed anything.

In future please feel free to raise an issue instead of starting a discussion if you feel something is broken or not working as expected.
